tests/deploy/k8s-cloudbeat-tests/requirements.yaml

```diff
@@ -0,0 +1,4 @@
+dependencies:
```

Comment:
Remove requirments.yml , move dependcies section to chart.yml and add a condition via value
https://helm.sh/docs/chart_best_practices/dependencies/#conditions-and-tags
https://stackoverflow.com/questions/54032974/helm-conditionally-install-subchart

Reply:
Unpack charts inside charts repo and control their toggle via the condition vars.
We can use it as a local custom sub chart instead of external chart dependency

Reply:
I think it is not argent to improve it during this PR.

It can be done during the following tasks
- https://github.com/elastic/security-team/issues/4717

or 
- https://github.com/elastic/security-team/issues/4713